indico/htdocs/js/indico/modules/registration/form/tpls/table.tpl.html

```diff
@@ -45,7 +45,7 @@
                     size="{{ elem.editoptions.size }}"
                     maxlength="{{ elem.editoptions.maxlength }}"
                     name="{{ elem.name }}"
-                    ng-required="elem.index == 'price' && !!item['billable']"
+                    ng-required="elem.index == 'price' && !!item['isBillable']"
```

Comment:
I don't know why it was done like that at the beginning in many places of the registration form Angular JS code and I'm scared to change it, but we could try and clean up some of the ugliest bits if we don't break anything.